### ISSUE TITLE
uniqid namespace false positive

### ISSUE THREAD

**Issue by azdanov** [kalessil/phpinspectionsea#825]
<img width="773" alt="screen shot 2018-02-26 at 16 35 50" src="https://user-images.githubusercontent.com/6123841/36675989-1fb8534e-1b13-11e8-8beb-057c8db412be.png">

Here's a code snippet:

```php
<?php

declare(strict_types=1);

namespace Acme;

use function uniqid;

/**
 * Class UniqId
 */
class UniqId
{
    /**
     * @return string
     */
    public function generateUniqId(): string
    {
        return uniqid('', true);
    }
}
```

**Comment by kalessil** [comment]
Looks very familiar, I'll fix it anyway. Thank you for reporting.
